types/three/src/materials/MeshPhysicalMaterial.d.ts

```diff
@@ -16,17 +16,29 @@ export interface MeshPhysicalMaterialParameters extends MeshStandardMaterialPara
 
     sheen?: number | undefined;
     sheenColor?: Color | undefined;
+    sheneColorMap?: Texture | null | undefined;
     sheenRoughness?: number | undefined;
+    sheneRoughnessMap?: Texture | null | undefined;
 
     transmission?: number | undefined;
     transmissionMap?: Texture | null | undefined;
+
+    thickness?: number | undefined;
+    thicknessMap?: Texture | null | undefined;
+
     attenuationDistance?: number | undefined;
     attenuationColor?: Color | undefined;
 
     specularIntensity?: number | undefined;
     specularColor?: Color | undefined;
     specularIntensityMap?: Texture | null | undefined;
     specularColorMap?: Texture | null | undefined;
+
+    iridescenceMap?: Texture | null;
+    iridescenceIOR?: number;
+    iridescence?: number;
+    iridescenceThicknessRange?: number[];
```

Comment:
this might be `[number, number]` instead of `number[]` ...? Is there any similar definitions around?

Reply:
thank you. I reflected your opinion.